app/src/main/res/values/strings.xml

```diff
@@ -330,4 +330,17 @@
     <string name="inbox_activity_messages">Messages</string>
     <string name="bookmarks_activity_saved">Saved</string>
     <string name="generic_error">An error has occurred</string>
+    <string name="lang_en" translatable="false">English</string>
```

Comment:
In the system language picker it defines the names for us. Can we access this at runtime instead of keeping it here?

Reply:
[The locales_config controls which languages are displayed in the language per app setting, so](https://stackoverflow.com/questions/36061116/get-language-name-in-that-language-from-language-code)

Not from there that I can tell but i did find another way